### PR TITLE
Update migrating-oauth-apps-to-github-apps.md

### PR DESCRIPTION
**Pull request by MelnikovAG**
These changes are to include the 'Delete OAuth App' step in 'Migrating OAuth Apps to GitHub Apps'

### Why:

Closes [issue link]

### Check off the following:

- [x] I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by Rob122783** [comment]
``
Duplicate of #

**Comment by SaintPatricksCoin** [comment]
> These changes are to include the 'Delete OAuth App' step in 'Migrating OAuth Apps to GitHub Apps'
> 
> ### Why:
> Closes [issue link]
> 
> ### What's being changed:
> ### Check off the following:
> * [x]  I have reviewed my changes in staging (look for the latest deployment event in your pull request's timeline, then click **View deployment**).
> * [x]  For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).
> 
> ### Writer impact (This section is for GitHub staff members only):
> * [ ]  This pull request impacts the contribution experience
>   
>   * [ ]  I have added the 'writer impact' label
>   * [ ]  I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

**Review by SaintPatricksCoin — COMMENTED** [review]
content/developers/apps/getting-started-with-apps/migrating-oauth-apps-to-github-apps.md
gh pr checkout 8749
MelnikovAG:123MwanjeMike-patch-2
https://github.com/github/docs/pull/8749#issue-958609965
.github/workflows/greetings.yml

**Review by Barabara343434 — COMMENTED** [review]
Harika

**Comment by Michaelstheshit** [comment]
Hey I m mike

**Comment by mom998** [comment]
Hello mike

File: content/developers/apps/getting-started-with-apps/migrating-oauth-apps-to-github-apps.md [hunk 1]
```diff
@@ -100,4 +101,8 @@ Once your GitHub App has been installed on a repository, you should remove any u
 
 ### Encourage users to revoke access to your OAuth app
 
-As your GitHub App installation base grows, consider encouraging your users to revoke access to the legacy OAuth integration. For more information, see [Authorizing OAuth Apps](/github/authenticating-to-github/keeping-your-account-and-data-secure/authorizing-oauth-apps)
+As your GitHub App installation base grows, consider encouraging your users to revoke access to the legacy OAuth integration. For more information, see "[Authorizing OAuth Apps](/github/authenticating-to-github/keeping-your-account-and-data-secure/authorizing-oauth-apps)."
+
+### Delete the OAuth App
+
+To avoid abuse of the OAuth App's credentials, consider deleting the OAuth App. This action will also revoke all of the OAuth App's remaining authorizations. For more information, see "[Deleting an OAuth App](/developers/apps/managing-oauth-apps/deleting-an-oauth-app)."
```
Comment (LEOF4U7) [inline content/developers/apps/getting-started-with-apps/migrating-oauth-apps-to-github-apps.md:108]: Necesito equipo (hadware) para adaptación de mejoras

Comment (mom998) [reply content/developers/apps/getting-started-with-apps/migrating-oauth-apps-to-github-apps.md:108]: Good

**Comment by mom998** [comment]
I'm need coins free in application sango please

File: content/developers/apps/getting-started-with-apps/migrating-oauth-apps-to-github-apps.md [hunk 2]
```diff
@@ -100,4 +101,8 @@ Once your GitHub App has been installed on a repository, you should remove any u
 
 ### Encourage users to revoke access to your OAuth app
 
-As your GitHub App installation base grows, consider encouraging your users to revoke access to the legacy OAuth integration. For more information, see [Authorizing OAuth Apps](/github/authenticating-to-github/keeping-your-account-and-data-secure/authorizing-oauth-apps)
+As your GitHub App installation base grows, consider encouraging your users to revoke access to the legacy OAuth integration. For more information, see "[Authorizing OAuth Apps](/github/authenticating-to-github/keeping-your-account-and-data-secure/authorizing-oauth-apps)."
```
Comment (sevrb) [inline content/developers/apps/getting-started-with-apps/migrating-oauth-apps-to-github-apps.md:104]: Yejg

Comment (timbo4678) [reply content/developers/apps/getting-started-with-apps/migrating-oauth-apps-to-github-apps.md:104]: I agree

**Pull request closed without merging**